libkbfs/journal_server.go

```diff
@@ -390,16 +396,24 @@ func (j *JournalServer) EnableExistingJournals(
 		// happen...
 		return err
 	}
+	close(journalCh)
+
+	for r := range journalCh {
```

Comment:
Well, I could read from it, but I couldn't update `j.tlfJournals` until waiting on `eg`, because that would trigger the same race.